week-1/implement/sum.js

```diff
@@ -0,0 +1,16 @@
+function sum(arr) {
+  let sumResult = 0;
+
+  for (const num of arr) {
+    if (typeof num === "number" && num !== "number") {
+      sumResult += num;
+    }
+  }
+
+  return sumResult;
+}
+
```

Comment:
Again no console.logs required to be pushed! 
Can you please use jest to test your functions before you push. Also I can't see anything regarding jest been pushed ?

Every file need file_name.test.js that can use jest to test your code.